_data/examples.json

```diff
@@ -193,37 +237,24 @@
       "title": "Trellis Area"
     }
   ],
-  "Layering": [
+  "Using Layer to Add Labels": [
```

Comment:
why capitalize add?

Reply:
It's a heading.

Reply:
I think this heading is inconsistent with the others. The heading are about what, now what for.

Reply:
Well, using layer to label is different from when you really layer graphics.  Plus, some people will be looking around to see how to add labels.